src/styles/index.scss

```diff
@@ -1,3 +1,155 @@
+/* #region generic */
+html {
+  font-family: "Roboto", sans-serif;
+  --main-accent: #060b35;
+  --secondary: #616070;
+  --blue-accent: #00acdc;
+  font-style: normal;
+}
+
 body {
   margin: 0;
 }
+
+/* #endregion*/
+/* #region card */
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: 200px;
+  justify-content: center;
+  align-items: center;
+  border: 1px solid black;
+
+  &-image {
+    width: 160px;
+    height: 134px;
+    flex-shrink: 0;
+
+    margin-top: 32px;
+  }
+
+  &-main-text {
+    color: var(--main-accent);
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    flex-grow: 1;
+    width: 166px;
+
+    margin-top: 40px;
+  }
+
+  &-subtitle-text {
+    align-self: flex-start;
+    color: var(--secondary);
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+
+    margin-top: 4px;
+    margin-left: 16px;
+    // margin-bottom: 0px;
```

Comment:
Remove redundant comments to keep your code clean

Reply:
Done